api/server.go

```diff
@@ -12,13 +12,16 @@ import (
 	"github.com/mattermost/mattermost-load-test-ng/performance"
 
 	"github.com/gorilla/mux"
+	"github.com/mattermost/mattermost-server/v5/mlog"
 )
 
 // api keeps track of the load-test API server state.
 type api struct {
 	agents       map[string]*loadtest.LoadTester
 	coordinators map[string]*coordinator.Coordinator
 	metrics      *performance.Metrics
+	clog         *mlog.Logger
+	alog         *mlog.Logger
```

Comment:
Would prefer them to be `coordLog` and `agentLog`. Somebody reading the codebase without any context would not understand quickly what they mean.

Reply:
Fair enough :)